testutil/testutil_test.go

```diff
@@ -1,17 +1,37 @@
 package testutil
 
 import (
-	"fmt"
-	"google.golang.org/protobuf/types/descriptorpb"
-	"path/filepath"
-	"runtime"
+	"os"
 	"testing"
+
+	"google.golang.org/protobuf/types/descriptorpb"
 )
 
 func TestDescriptorFromProtoSource_CustomProtoPaths(t *testing.T) {
-	_, thisFilePath, _, _ := runtime.Caller(0)
+	dirName := os.TempDir() + string(os.PathSeparator) + "api-linter-fake-dir"
```

Comment:
This folder name is not random -- it may conflict -- use `ioutil.TempDir` instead.